### ISSUE TITLE
clients stutter with sleep < -bufferDuration/2 ("onResync" on server)

### ISSUE THREAD

**Issue by yannok** [badaix/snapcast#326]
I face frequent simultaneous short playback interruptions on all my snapcast clients (all raspberry 3 with up-to-date raspbian jessie).
The server is also a Rpi3 with mopidy+snapserver.
Server options:
`SNAPSERVER_OPTS="--codec pcm --sampleformat 48000:16:2"`
Client options:
`SNAPCLIENT_OPTS="-d -p 6600"`

Server output when an interruption occurs:
```
2017-12-06 17-06-10 [out] onStateChanged (default): 1
2017-12-06 17-06-10 [out] onResync (default): 282ms
2017-12-06 17-06-10 [out] onStateChanged (default): 2
```

Client output when an interruption occurs: (launched manually to get the output, not as a service)
```
2017-12-06 17-06-10 [out] Chunk: 0      0       0       0       500     119
2017-12-06 17-06-11 [out] Chunk: 0      0       0       0       500     89
2017-12-06 17-06-11 [out] age > 200: -282
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -282 < -15, sleep: -282
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -282 < -15, sleep: -252
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -252 < -15, sleep: -222
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -222 < -15, sleep: -192
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -192 < -15, sleep: -162
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -162 < -15, sleep: -132
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -132 < -15, sleep: -102
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -102 < -15, sleep: -72
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -72 < -15, sleep: -42
2017-12-06 17-06-11 [out] sleep < -bufferDuration/2: -42 < -15, sleep: -12
2017-12-06 17-06-11 [out] Sleep -12, age: -11, bufferDuration: 30
2017-12-06 17-06-11 [out] Sleep -11, age: -12, bufferDuration: 30
2017-12-06 17-06-11 [out] Sleep -11, age: -11, bufferDuration: 30
2017-12-06 17-06-11 [out] Sleep -10, age: -10, bufferDuration: 30
2017-12-06 17-06-12 [out] Chunk: -106   -106    -106    -106    1       89
```

I checked the RAM and CPU load on both clients and server and they are fine.
Clients and server are on the same LAN (no wifi).

Any suggestion to debug this issue? I love this software but unfortunately, I can't make it work correctly...

Thanks

**Comment by christf** [comment]
I am seeing the same and have prepared a tcpdump. maybe it is of help getting to the bottom of this. I have not done much with tcp yet.

```
18:08:37.830668 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11285431, win 4606, options [nop,nop,TS val 677040754 ecr 1239388919], length 0
18:08:37.831085 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11285431:11286879, ack 5804, win 235, options [nop,nop,TS val 1239388922 ecr 677040751], length 1448
18:08:37.838352 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11311495:11312943, ack 5804, win 235, options [nop,nop,TS val 1239388922 ecr 677040751], length 1448
18:08:37.838362 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11312943:11314391, ack 5804, win 235, options [nop,nop,TS val 1239388922 ecr 677040751], length 1448
18:08:37.838363 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11314391:11314889, ack 5804, win 235, options [nop,nop,TS val 1239388922 ecr 677040751], length 498
18:08:37.844558 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11314889, win 4606, options [nop,nop,TS val 677040768 ecr 1239388922], length 0
18:08:38.122571 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [P.], seq 5804:5838, ack 11314889, win 4606, options [nop,nop,TS val 677041046 ecr 1239388922], length 34
18:08:38.128416 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11314889:11314923, ack 5838, win 235, options [nop,nop,TS val 1239388996 ecr 677041046], length 34
18:08:38.128672 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11314923, win 4606, options [nop,nop,TS val 677041052 ecr 1239388996], length 0
18:08:40.520675 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11314923:11316371, ack 5838, win 235, options [nop,nop,TS val 1239389594 ecr 677041052], length 1448
18:08:40.521022 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11316371, win 4591, options [nop,nop,TS val 677043444 ecr 1239389594], length 0
18:08:40.520716 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11316371:11317819, ack 5838, win 235, options [nop,nop,TS val 1239389594 ecr 677041052], length 1448
18:08:40.521420 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11317819, win 4577, options [nop,nop,TS val 677043445 ecr 1239389594], length 0
18:08:40.520781 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11317819:11319267, ack 5838, win 235, options [nop,nop,TS val 1239389594 ecr 677041052], length 1448
18:08:40.521727 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11319267, win 4563, options [nop,nop,TS val 677043445 ecr 1239389594], length 0
18:08:40.522062 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11319267:11320715, ack 5838, win 235, options [nop,nop,TS val 1239389594 ecr 677041052], length 1448
18:08:40.526276 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11330851:11332299, ack 5838, win 235, options [nop,nop,TS val 1239389595 ecr 677043444], length 1448
18:08:40.526601 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11332299, win 4437, options [nop,nop,TS val 677043450 ecr 1239389595], length 0
18:08:40.526333 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11332299:11333747, ack 5838, win 235, options [nop,nop,TS val 1239389595 ecr 677043445], length 1448
18:08:40.527061 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11333747, win 4423, options [nop,nop,TS val 677043450 ecr 1239389595], length 0
18:08:40.526371 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11333747:11335195, ack 5838, win 235, options [nop,nop,TS val 1239389595 ecr 677043445], length 1448
18:08:40.527394 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11335195, win 4409, options [nop,nop,TS val 677043451 ecr 1239389595], length 0
18:08:40.526399 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11335195:11336643, ack 5838, win 235, options [nop,nop,TS val 1239389596 ecr 677043446], length 1448
18:08:40.541466 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11375739:11377187, ack 5838, win 235, options [nop,nop,TS val 1239389599 ecr 677043460], length 1448
18:08:40.820209 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [P.], seq 5838:5872, ack 11421483, win 4606, options [nop,nop,TS val 677043744 ecr 1239389599], length 34
18:08:40.826126 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11421483:11421517, ack 5872, win 235, options [nop,nop,TS val 1239389670 ecr 677043744], length 34
18:08:40.826430 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11421517, win 4606, options [nop,nop,TS val 677043750 ecr 1239389670], length 0
18:08:43.308701 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11421517:11422965, ack 5872, win 235, options [nop,nop,TS val 1239390291 ecr 677043750], length 1448
18:08:43.309018 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11422965, win 4591, options [nop,nop,TS val 677046232 ecr 1239390291], length 0
18:08:43.308759 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11422965:11424413, ack 5872, win 235, options [nop,nop,TS val 1239390291 ecr 677043750], length 1448
18:08:43.309346 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11424413, win 4577, options [nop,nop,TS val 677046233 ecr 1239390291], length 0
18:08:43.308786 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11424413:11425861, ack 5872, win 235, options [nop,nop,TS val 1239390291 ecr 677043750], length 1448
18:08:43.309618 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11425861, win 4563, options [nop,nop,TS val 677046233 ecr 1239390291], length 0
18:08:43.308809 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11425861:11427309, ack 5872, win 235, options [nop,nop,TS val 1239390291 ecr 677043750], length 1448
18:08:43.333346 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11508397:11509845, ack 5872, win 235, options [nop,nop,TS val 1239390295 ecr 677046244], length 1448
18:08:43.333352 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11509845:11511293, ack 5872, win 235, options [nop,nop,TS val 1239390295 ecr 677046244], length 1448
18:08:43.333355 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11511293:11512741, ack 5872, win 235, options [nop,nop,TS val 1239390295 ecr 677046244], length 1448
18:08:43.333358 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11512741:11514189, ack 5872, win 235, options [nop,nop,TS val 1239390295 ecr 677046244], length 1448
18:08:43.605118 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [P.], seq 5872:5906, ack 11526986, win 4606, options [nop,nop,TS val 677046529 ecr 1239390295], length 34
18:08:43.611129 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11526986:11527020, ack 5906, win 235, options [nop,nop,TS val 1239390366 ecr 677046529], length 34
18:08:43.611374 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11527020, win 4606, options [nop,nop,TS val 677046535 ecr 1239390366], length 0
18:08:45.999221 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11527020:11528468, ack 5906, win 235, options [nop,nop,TS val 1239390964 ecr 677046535], length 1448
18:08:45.999558 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11528468, win 4591, options [nop,nop,TS val 677048923 ecr 1239390964], length 0
18:08:45.999259 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11528468:11529916, ack 5906, win 235, options [nop,nop,TS val 1239390964 ecr 677046535], length 1448
18:08:45.999963 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11529916, win 4577, options [nop,nop,TS val 677048923 ecr 1239390964], length 0
18:08:45.999287 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11529916:11531364, ack 5906, win 235, options [nop,nop,TS val 1239390964 ecr 677046535], length 1448
18:08:46.000270 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11531364, win 4563, options [nop,nop,TS val 677048924 ecr 1239390964], length 0
18:08:45.999313 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11531364:11532812, ack 5906, win 235, options [nop,nop,TS val 1239390964 ecr 677046535], length 1448
18:08:46.004084 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11538604:11540052, ack 5906, win 235, options [nop,nop,TS val 1239390964 ecr 677046535], length 1448
18:08:46.004395 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11540052, win 4479, options [nop,nop,TS val 677048928 ecr 1239390964], length 0
18:08:46.008557 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11560324:11561772, ack 5906, win 235, options [nop,nop,TS val 1239390965 ecr 677048926], length 1448
18:08:46.013153 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11577700:11579148, ack 5906, win 235, options [nop,nop,TS val 1239390966 ecr 677048930], length 1448
18:08:46.013189 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11579148:11580596, ack 5906, win 235, options [nop,nop,TS val 1239390966 ecr 677048930], length 1448
18:08:46.013232 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11580596:11582044, ack 5906, win 235, options [nop,nop,TS val 1239390966 ecr 677048930], length 1448
18:08:46.013264 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11582044:11583492, ack 5906, win 235, options [nop,nop,TS val 1239390967 ecr 677048931], length 1448
18:08:46.014483 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11583492:11584940, ack 5906, win 235, options [nop,nop,TS val 1239390967 ecr 677048931], length 1448
18:08:46.014518 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11584940:11586388, ack 5906, win 235, options [nop,nop,TS val 1239390967 ecr 677048931], length 1448
18:08:46.014556 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11586388:11587836, ack 5906, win 235, options [nop,nop,TS val 1239390967 ecr 677048931], length 1448
18:08:46.014592 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11587836:11589284, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048933], length 1448
18:08:46.017470 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11589284:11590732, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048933], length 1448
18:08:46.017507 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11590732:11592180, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048933], length 1448
18:08:46.017532 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11592180:11593628, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048933], length 1448
18:08:46.017560 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11593628:11595076, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048933], length 1448
18:08:46.018118 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11595076:11596524, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048933], length 1448
18:08:46.018152 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11596524:11597972, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048933], length 1448
18:08:46.025439 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11613900:11615348, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048934], length 1448
18:08:46.025448 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11615348:11616796, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048934], length 1448
18:08:46.025453 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11616796:11618244, ack 5906, win 235, options [nop,nop,TS val 1239390968 ecr 677048934], length 1448
18:08:46.037267 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11635133, win 4606, options [nop,nop,TS val 677048961 ecr 1239390968], length 0
18:08:46.309530 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [P.], seq 5906:5940, ack 11635133, win 4606, options [nop,nop,TS val 677049233 ecr 1239390968], length 34
18:08:46.313748 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11635133:11635167, ack 5940, win 235, options [nop,nop,TS val 1239391042 ecr 677049233], length 34
18:08:46.314008 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11635167, win 4606, options [nop,nop,TS val 677049237 ecr 1239391042], length 0
18:08:48.692897 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11635167:11636615, ack 5940, win 235, options [nop,nop,TS val 1239391637 ecr 677049237], length 1448
18:08:48.693243 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11636615, win 4591, options [nop,nop,TS val 677051617 ecr 1239391637], length 0
18:08:48.692946 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11636615:11638063, ack 5940, win 235, options [nop,nop,TS val 1239391637 ecr 677049237], length 1448
18:08:48.693609 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11638063, win 4577, options [nop,nop,TS val 677051617 ecr 1239391637], length 0
18:08:48.692977 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11638063:11639511, ack 5940, win 235, options [nop,nop,TS val 1239391637 ecr 677049237], length 1448
18:08:48.693914 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11639511, win 4563, options [nop,nop,TS val 677051617 ecr 1239391637], length 0
18:08:48.693001 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11639511:11640959, ack 5940, win 235, options [nop,nop,TS val 1239391637 ecr 677049237], length 1448
18:08:48.708217 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11704671:11706119, ack 5940, win 235, options [nop,nop,TS val 1239391640 ecr 677051625], length 1448
18:08:48.708269 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11706119:11707567, ack 5940, win 235, options [nop,nop,TS val 1239391640 ecr 677051625], length 1448
18:08:48.712695 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11709015, win 4606, options [nop,nop,TS val 677051636 ecr 1239391639], length 0
18:08:48.717244 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11709015:11710463, ack 5940, win 235, options [nop,nop,TS val 1239391640 ecr 677051625], length 1448
18:08:48.717278 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11710463:11711911, ack 5940, win 235, options [nop,nop,TS val 1239391640 ecr 677051625], length 1448
18:08:48.717700 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11711911:11713359, ack 5940, win 235, options [nop,nop,TS val 1239391640 ecr 677051625], length 1448
18:08:48.717749 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11713359:11714807, ack 5940, win 235, options [nop,nop,TS val 1239391640 ecr 677051625], length 1448
18:08:48.717778 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11714807:11716255, ack 5940, win 235, options [nop,nop,TS val 1239391640 ecr 677051625], length 1448
18:08:48.720806 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11729287:11730735, ack 5940, win 235, options [nop,nop,TS val 1239391641 ecr 677051626], length 1448
18:08:49.007180 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [P.], seq 5940:5974, ack 11739507, win 4606, options [nop,nop,TS val 677051931 ecr 1239391644], length 34
18:08:49.013062 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11739507:11739541, ack 5974, win 235, options [nop,nop,TS val 1239391717 ecr 677051931], length 34
18:08:49.013317 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11739541, win 4606, options [nop,nop,TS val 677051937 ecr 1239391717], length 0
18:08:51.483971 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11739541:11740989, ack 5974, win 235, options [nop,nop,TS val 1239392335 ecr 677051937], length 1448
18:08:51.484317 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11740989, win 4591, options [nop,nop,TS val 677054408 ecr 1239392335], length 0
18:08:51.484014 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11740989:11742437, ack 5974, win 235, options [nop,nop,TS val 1239392335 ecr 677051937], length 1448
18:08:51.484683 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11742437, win 4577, options [nop,nop,TS val 677054408 ecr 1239392335], length 0
18:08:51.484050 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11742437:11743885, ack 5974, win 235, options [nop,nop,TS val 1239392335 ecr 677051937], length 1448
18:08:51.484992 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11743885, win 4563, options [nop,nop,TS val 677054408 ecr 1239392335], length 0
18:08:51.484077 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11743885:11745333, ack 5974, win 235, options [nop,nop,TS val 1239392335 ecr 677051937], length 1448
18:08:51.488603 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11751125:11752573, ack 5974, win 235, options [nop,nop,TS val 1239392335 ecr 677051937], length 1448
18:08:51.488869 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11752573, win 4591, options [nop,nop,TS val 677054412 ecr 1239392335], length 0
18:08:51.510735 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11843797:11845245, ack 5974, win 235, options [nop,nop,TS val 1239392340 ecr 677054424], length 1448
18:08:51.510775 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11845245:11845533, ack 5974, win 235, options [nop,nop,TS val 1239392340 ecr 677054424], length 288
18:08:51.511101 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11845533, win 4606, options [nop,nop,TS val 677054435 ecr 1239392340], length 0
18:08:51.779765 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [P.], seq 5974:6008, ack 11845533, win 4606, options [nop,nop,TS val 677054703 ecr 1239392340], length 34
18:08:51.785704 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11845533:11845567, ack 6008, win 235, options [nop,nop,TS val 1239392410 ecr 677054703], length 34
18:08:51.785960 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11845567, win 4606, options [nop,nop,TS val 677054709 ecr 1239392410], length 0
18:08:54.170626 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11845567:11847015, ack 6008, win 235, options [nop,nop,TS val 1239393007 ecr 677054709], length 1448
18:08:54.170921 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11847015, win 4591, options [nop,nop,TS val 677057094 ecr 1239393007], length 0
18:08:54.170665 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11847015:11848463, ack 6008, win 235, options [nop,nop,TS val 1239393007 ecr 677054709], length 1448
18:08:54.171291 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11848463, win 4577, options [nop,nop,TS val 677057095 ecr 1239393007], length 0
18:08:54.170690 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11848463:11849911, ack 6008, win 235, options [nop,nop,TS val 1239393007 ecr 677054709], length 1448
18:08:54.171595 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11849911, win 4563, options [nop,nop,TS val 677057095 ecr 1239393007], length 0
18:08:54.170713 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11849911:11851359, ack 6008, win 235, options [nop,nop,TS val 1239393007 ecr 677054709], length 1448
18:08:54.187961 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11897695:11899143, ack 6008, win 235, options [nop,nop,TS val 1239393010 ecr 677057100], length 1448
18:08:54.188003 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11899143:11900591, ack 6008, win 235, options [nop,nop,TS val 1239393010 ecr 677057100], length 1448
18:08:54.189143 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11900591:11902039, ack 6008, win 235, options [nop,nop,TS val 1239393010 ecr 677057100], length 1448
18:08:54.194511 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11923759:11925207, ack 6008, win 235, options [nop,nop,TS val 1239393010 ecr 677057102], length 1448
18:08:54.194593 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11925207:11926655, ack 6008, win 235, options [nop,nop,TS val 1239393011 ecr 677057105], length 1448
18:08:54.194621 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11926655:11928103, ack 6008, win 235, options [nop,nop,TS val 1239393011 ecr 677057105], length 1448
18:08:54.194651 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11928103:11929551, ack 6008, win 235, options [nop,nop,TS val 1239393011 ecr 677057105], length 1448
18:08:54.195247 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11929551:11930999, ack 6008, win 235, options [nop,nop,TS val 1239393011 ecr 677057105], length 1448
18:08:54.195288 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11930999:11932447, ack 6008, win 235, options [nop,nop,TS val 1239393011 ecr 677057105], length 1448
18:08:54.195317 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11932447:11933895, ack 6008, win 235, options [nop,nop,TS val 1239393011 ecr 677057105], length 1448
18:08:54.202194 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11952719:11954167, ack 6008, win 235, options [nop,nop,TS val 1239393013 ecr 677057115], length 1448
18:08:54.202241 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11954167:11954705, ack 6008, win 235, options [nop,nop,TS val 1239393013 ecr 677057115], length 538
18:08:54.203044 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11954705, win 4606, options [nop,nop,TS val 677057127 ecr 1239393013], length 0
18:08:54.485331 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [P.], seq 6008:6042, ack 11954705, win 4606, options [nop,nop,TS val 677057409 ecr 1239393013], length 34
18:08:54.494588 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 11954705:11954739, ack 6042, win 235, options [nop,nop,TS val 1239393087 ecr 677057409], length 34
18:08:54.494845 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11954739, win 4606, options [nop,nop,TS val 677057418 ecr 1239393087], length 0
18:08:56.871638 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11954739:11956187, ack 6042, win 235, options [nop,nop,TS val 1239393682 ecr 677057418], length 1448
18:08:56.871965 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11956187, win 4591, options [nop,nop,TS val 677059795 ecr 1239393682], length 0
18:08:56.871677 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11956187:11957635, ack 6042, win 235, options [nop,nop,TS val 1239393682 ecr 677057418], length 1448
18:08:56.872325 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11957635, win 4577, options [nop,nop,TS val 677059796 ecr 1239393682], length 0
18:08:56.871703 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11957635:11959083, ack 6042, win 235, options [nop,nop,TS val 1239393682 ecr 677057418], length 1448
18:08:56.872627 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 11959083, win 4563, options [nop,nop,TS val 677059796 ecr 1239393682], length 0
18:08:56.871729 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11959083:11960531, ack 6042, win 235, options [nop,nop,TS val 1239393682 ecr 677057418], length 1448
18:08:56.875686 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 11966323:11967771, ack 6042, win 235, options [nop,nop,TS val 1239393682 ecr 677057418], length 1448
18:08:56.889162 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 12030035:12031483, ack 6042, win 235, options [nop,nop,TS val 1239393685 ecr 677059803], length 1448
18:08:56.890413 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 12035827:12037275, ack 6042, win 235, options [nop,nop,TS val 1239393686 ecr 677059807], length 1448
18:08:56.891343 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 12041619, win 4606, options [nop,nop,TS val 677059815 ecr 1239393686], length 0
18:08:56.892432 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [.], seq 12041619:12043067, ack 6042, win 235, options [nop,nop,TS val 1239393687 ecr 677059813], length 1448
18:08:56.898033 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 12064219, win 4606, options [nop,nop,TS val 677059822 ecr 1239393687], length 0
18:08:57.170189 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [P.], seq 6042:6076, ack 12064219, win 4606, options [nop,nop,TS val 677060094 ecr 1239393687], length 34
18:08:57.178241 IP 192.168.13.223.1704 > 192.168.13.135.33486: Flags [P.], seq 12064219:12064253, ack 6076, win 235, options [nop,nop,TS val 1239393757 ecr 677060094], length 34
18:08:57.178589 IP 192.168.13.135.33486 > 192.168.13.223.1704: Flags [.], ack 12064253, win 4606, options [nop,nop,TS val 677060102 ecr 1239393757], length 0
```

**Comment by christf** [comment]
The root cause is #430 and fixed with #432.

**Comment by badaix** [comment]
fixed in v0.19.0
